### PR TITLE
pygments: cleanup, update to version 2.14.0

### PR DESCRIPTION
**Pull request by OscarL**
Move the PROVIDES_* inside the loop, fix script suffixes for 3.10.

Install from the wheel file instead of using the tarball for faster package buids (1.1 MB vs 4.4 MB file download).

### CODE REVIEW

**Comment by korli** [comment]
Usually we keep the tarball because the tests, the manpage, the docs are otherwise not provided.

**Comment by korli** [comment]
It looks good to me. You might install the manpage, if it's missing.

File: dev-python/pygments/pygments-2.14.0.recipe [hunk 1]
```diff
@@ -50,32 +53,36 @@ BUILD_PREREQUIRES="$BUILD_PREREQUIRES
...
 			--root=/ --prefix=$prefix
 
-		if [ $pythonPackage = python38 ]; then
-			mv $binDir/pygmentize $binDir/pygmentize3.8
+		mkdir -p $manDir/man1
+		cp doc/pygmentize.1 $manDir/man1
+
+		if [ "$pythonVersion" = "$commandSuffix" ]; then
+			for f in $binDir/*; do
+				mv $f ${f}$commandSuffix
```
Comment (Begasus) [inline dev-python/pygments/pygments-2.14.0.recipe:79]: This is somehow broken? 
```
Warning: POLICY ERROR: no matching provides "cmd:pygmentize" for "bin/pygmentize"
```

Comment (Begasus) [reply dev-python/pygments/pygments-2.14.0.recipe:79]: Fixed in: https://github.com/haikuports/haikuports/pull/9852

**Pull request merged**